ALiAS_FAQs.md

```diff
@@ -0,0 +1,47 @@
+                                                FAQs
+Q1. What is ASET AliAS?
+Ans. It’s not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the Open Source. We community members help students with their doubts pertaining tech, conduct workshops, webinars, have meet-ups (online and in-person) and most important get you connected.
+
+Q2. Why should I join ALiAS?
+Ans. ALiAS is by far, the largest open-source community in Amity University, we are
+open to developers not only inside the campus, but also from various places. We have got assistance
+from alumni and people from tech, near and far. We have got presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
+
+Q3. How is the community of AliAS?
+And. We have got the most active developers in the campus. You will find people in nearly all domains,
+Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people,
+etc. And best of all, we have got a great reach for people and further development.
+
+Q4. How should I join AliAS?
+And. Simple. Go and register yourself in http://asetalias.in/ . One of our developers will get you joined
+to the group.
+
+Q5. Having joined, where should I start?
+And. We request you to drop a message, stating your name, course duration and your skills, if any. Also
```

Comment:
```suggestion
Ans. We request you to drop a message, stating your name, course duration, and your skills. Also
```
Bro, `if any` is rude lol

Reply:
Rude? Ever saw any freshers batch in Amity? Nothing can be more disappointing than that. People can't even understand simple print statements! Although, it had a satirical remark, they literally don't know anything. But okay. Accepted XD

Reply:
I never said it was wrong, just rude. If someone is bad at something, doesn't mean you have to confront them about it :rofl:  especially when you're introducing them to a new community.

Reply:
Also, everyone starts like that

Reply:
Being humble every time, everywhere is everything in an open-source community. No one is born with the knowledge of programming, no one understands at the same rate. Hence, we cannot, should not and will not pass judgment on the basis of skills, experience, age, and especially if there are freshers.

> Ever saw any freshers batch in Amity? Nothing can be more disappointing than that. People can't even understand simple print statements!

Well, that means I would be pretty disappointing then. Before joining Amity, I was the technical head of my school's technical society even when I didn't opt for Computer Science as my 5th. I won many awards for website design. Never wrote a single line of code. I had a hard time getting over the barrier to understand basic shit. This wouldn't have been possible if Shyam bhaiya would have been disappointed in me before giving me a single chance. Give the people a chance that they deserve, remain humble, be positive. Carry on...

Reply:
Okay... :pray: